src/pages/my_page/user_profile/ProfilePictureUploader.tsx

```diff
@@ -0,0 +1,110 @@
+import AddAPhotoIcon from '@mui/icons-material/AddAPhoto';
+import CancelIcon from '@mui/icons-material/Cancel';
+import { Box, IconButton, Skeleton, Typography } from '@mui/material';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { ChangeEvent } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { fetchProfilePicture, uploadProfilePicture } from '../../../api/cristinApi';
+import { setNotification } from '../../../redux/notificationSlice';
+import { dataTestId } from '../../../utils/dataTestIds';
+
+interface ProfilePictureUploaderProps {
+  id: string;
+}
+
+export const ProfilePictureUploader = ({ id }: ProfilePictureUploaderProps) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  const profilePictureQuery = useQuery({
+    queryKey: ['picture', id],
+    queryFn: () => fetchProfilePicture(id),
+    meta: { errorMessage: false },
+    retry: false,
+  });
+
+  const mutateProfilePicture = useMutation({
+    mutationFn: (base64String: string) => uploadProfilePicture(id, base64String),
+    onSuccess: async () => {
+      await profilePictureQuery.refetch();
+      dispatch(setNotification({ message: t('feedback.success.update_profile_photo'), variant: 'success' }));
+    },
+    onError: () => {
+      dispatch(setNotification({ message: t('feedback.error.update_profile_photo'), variant: 'error' }));
+    },
+  });
+
+  const profilePictureString = profilePictureQuery.isSuccess
+    ? `data:image/jpeg;base64,${profilePictureQuery.data?.base64Data}`
```

Comment:
hvis vi vet det er success, vet vi også at den har `data`, eller?
```suggestion
    ? `data:image/jpeg;base64,${profilePictureQuery.data.base64Data}`
```